Formula/postgresql@12.rb

```diff
@@ -115,16 +115,16 @@ def plist
         <array>
           <string>#{opt_bin}/postgres</string>
           <string>-D</string>
-          <string>#{var}/postgres</string>
+          <string>#{var}/#{name}</string>
```

Comment:
These directories also need to be created in `post_install` then, and updated in the caveat.